docs/api/error.rst

```diff
@@ -25,6 +22,8 @@ To retrieve metadata on the datasets known to the datastore.
       -  ``last_parsed`` - last date resource was successfully parsed
       -  ``num_of_activities`` number of activities datastore has
          successfully parsed and stored
+-   ``/api/1/about/datasets/nest`` retrieves the list of datasets currently
```

Comment:
Name now changes

Reply:
actually if we're not documenting it this should go entirely